Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.cpp

```diff
@@ -276,7 +276,7 @@ void InjectedBundle::setUserStyleSheetLocation(const String& location)
 void InjectedBundle::removeAllWebNotificationPermissions(WebPage* page)
 {
 #if ENABLE(NOTIFICATIONS)
-    page->notificationPermissionRequestManager()->removeAllPermissionsForTesting();
+    RefPtr { page }->notificationPermissionRequestManager()->removeAllPermissionsForTesting();
```

Comment:
This is not right, `page` is safe here since this is a parameter and parameters are protected at the call sites.

The static analysis is about `notificationPermissionRequestManager()`. We should introduce a `WebPage::protectedNotificationPermissionRequestManager()` and call it.